sample-notificator-app/src/main/groovy/life/qbic/samplenotificator/SampleNotificatorEntryPoint.groovy

```diff
@@ -0,0 +1,29 @@
+package life.qbic.samplenotificator
+
+import groovy.util.logging.Log4j2
+import life.qbic.samplenotificator.cli.CommandLineParser
+import life.qbic.samplenotificator.cli.NotificatorCommandLineOptions
+
+/**
+ * <h1>Entry point for SampleNotificator service</h1>
+ *
+ * <p>Starts the service</p>
+ *
+ * @since 1.0.0
+ *
+*/
+@Log4j2
+class SampleNotificatorEntryPoint {
+
+    public static void main(String[] args) {
+        try {
+            NotificatorCommandLineOptions commandLineParameters =
+                    CommandLineParser.parseAndVerifyCommandLineParameters(args)
+
+            NotificatorApp app = new NotificatorApp(commandLineParameters)
+            app.run()
```

Comment:
You could also use the [QBiC classes](https://github.com/qbicsoftware/core-utils-lib/tree/master/src/main/java/life/qbic/cli) here.